conda-store-server/tests/test_actions.py

```diff
@@ -155,6 +164,7 @@ def test_remove_conda_prefix(tmp_path, simple_conda_lock):
     assert not conda_prefix.exists()
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="permissions are not supported on Windows")
```

Comment:
Use the same ON_WIN check here.

Reply:
I'll do the other way around and use the win32 check instead of the constant everywhere.